charts/drax/Chart.yaml

```diff
@@ -5,7 +5,7 @@ type: application
 version: 11.0.2
 dependencies:
   - name: common
-    version: 0.3.9
+    version: 0.4.0
```

Comment:
This version should be bumped by renovate (after that common chart is actually released).

Reply:
This change is also the main reason that the CI tests are failing btw. The 0.4.0 release doesn't exist yet, so the dependency can't be downloaded yet within the CI...